README.md

```diff
@@ -0,0 +1,98 @@
+# gramhopper
+A bot platform for automatic responses based on various triggers.
+
+## Install
+Install the latest version:
+
+```bash
+$ pip install gramhopper
+```
+
+## Configure
+
+### Create a bot
+1. [Create a bot using BotFather](https://core.telegram.org/bots#6-botfather).
+2. Save the received token in `~/.gramphopper/token.txt`.
+3. Enable receiving messages from groups for the bot by sending `/setprivacy` to BotFather,
+ then select the new bot and click `Disable`. 
+
+### Enable nicknames for user-based rules
+To use user nicknames, `~/.gramphopper/users.json` should contain an object in which the keys 
+are nicknames and the values are user IDs, for example:
+```json
```

Comment:
Maybe we should add a quick guidance \ screenshot on how do you get the IDs of the users.